infra/image/build.sh

```diff
@@ -88,6 +88,17 @@ if [ "${deploy_server}" == "Y" ]
 then
     deployed=false
 
+    # Set path to ansible-freeipa roles
+    [ -z "${ANSIBLE_ROLES_PATH:-""}" ] && export ANSIBLE_ROLES_PATH="${TOPDIR}/roles"
+
+    # Install collection containers.podman if not available
+    if [ -z "$(ansbile-galaxy collection list containers.podman)" ]
```

Comment:
There is a typo "ansbile-galaxy" instead of "ansible-galaxy".

Reply:
I've been bitten by this one. Fixed.